python/settings/user_settings.py

```diff
@@ -134,12 +167,13 @@ def retrieve(self, name, default=None, scope=SCOPE_GLOBAL):
         self.__fw.log_debug("User Settings Manager: Retrieving %s" % full_name)
 
         try:
-            raw_value = sanitize_qt(self.__settings.value(full_name))
```

Comment:
Settings were already sanitized on their way in, so no need to sanitize on their way out.